docs/source/get_submission_invitations.rst

```diff
@@ -1,101 +0,0 @@
-Getting Submissions
-========================================
-
-All the Invitation Ids for Submissions can be retrieved like this::
-
-    >>> import openreview
-    >>> c = openreview.Client(baseurl='https://api.openreview.net')
-    >>> invitations = openreview.tools.get_submission_invitations(c)
```

Comment:
@nlbahy do you mind removing the function `get_submission_invitations` from the tools file?